src/assets/styles/overrides.scss

```diff
@@ -81,3 +81,22 @@
   color: $app-red !important;
   caret-color: $app-red !important;
 }
+
+.filing-item__actions {
```

Comment:
This file should be global overrides, so should not contain component-specific classes.

Maybe I misunderstand (or didn't explain well)... I am in favour of `expand-btn` and `menu-btn` classes in here -- or reuse some existing (or new) classes in this file for your component, but don't make anything in here specific to "filing-item" for example.

Overrides in this file should directly apply to v- components.

If this can't be done in your scenario then please revert this code -- sorry.

Reply:
Either way, I think your latest css/styles is cleaner than before - great!

Reply:
Yes, I know. I see that we always do it for that button, not only for this case. So I think it should be a global override.